### PR TITLE
PHI v2 

### PR DESCRIPTION
**Pull request by shendel**
## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/swaponline/MultiCurrencyWallet/blob/master/docs/CONTRIBUTING.md) guide
- [ ] Good naming (as clear and simple as possible)
- [ ] Correct behavior if external API endpoints are down, return 404, 504 (or no answer), 401 errors (ddos simulation)
- [ ] I tested desktop/mobile resolution
- [ ] I tested light/dark theme
- [ ] I tested different languages
- [ ] I checked the functionality once again (**AFFECT MONEY**)
- [ ] I checked the work on the Testnet
- [ ] I checked the work on the Mainnet
- [ ] I checked the work in the plugin
- [ ] I checked the **PR** once again

## Tests

Please start auto tests as follows:

- add a label <ins>swap test</ins> to start swap tests
- add a label <ins>withdraw test</ins> to start withdraw tests

You can skip these tests if you completely sure that your changes aren't related to this functional

### Original issue

#

### CODE REVIEW

**Comment by Phinetwork** [comment]
Looks like it's coming along good. What would you say is the ETA on PHI Network v2 integration

**Comment by shendel** [comment]
> Looks like it's coming along good. What would you say is the ETA on PHI Network v2 integration

Will be ready by the end of this week

**Comment by Phinetwork** [comment]
> > Looks like it's coming along good. What would you say is the ETA on PHI Network v2 integration
> 
> 
> 
> Will be ready by the end of this week

Amazing great news. thanks for the update.

File: src/front/shared/helpers/constants/TOKEN_STANDARDS.ts [hunk 1]
```diff
@@ -118,8 +118,18 @@ const STANDARDS = {
     standard: 'phi20',
     value: 'phi20',
     currency: 'phi',
-    explorerApi: config.api?.aurorascan || '',
-    explorerApiKey: config.api?.phi_ApiKey || '',
+    explorerApi: config.api?.aurorascan || '', // ???
```
Comment (VitaliyShulik) [inline src/front/shared/helpers/constants/TOKEN_STANDARDS.ts:121]: здесь для etherscan-like explorer инфа, чтобы можно транзакции фетчить вроде.

File: src/front/shared/pages/Exchange/QuickSwap/index.tsx [hunk 2]
```diff
@@ -686,6 +686,9 @@ class QuickSwap extends PureComponent<IUniversalObj, ComponentState> {
     )
 
     if (!pairAddress) {
+      console.log('>>> LIQUIDITY_SOURCE_DATA', LIQUIDITY_SOURCE_DATA)
```
Comment (VitaliyShulik) [inline src/front/shared/pages/Exchange/QuickSwap/index.tsx:689]: лучше убрать

**Review by VitaliyShulik — APPROVED** [review]

File: src/front/shared/pages/Exchange/QuickSwap/index.tsx (hunk 2 above)
Comment (shendel) [reply src/front/shared/pages/Exchange/QuickSwap/index.tsx]: забыл почистить

**Pull request merged**